app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2 class='login_header'>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", :class => 'input' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", :class => 'input' %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Log in", :class => 'input submit' %>
+  </div>
+<% end %>
+
+<div class='links-cnt'>
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] The login form should be centered from the provided layout, but yours is aligned to the left. Kindly fix that